packages/toolkit/package.json

```diff
@@ -38,12 +37,15 @@
     "css-loader": "^6.7.1",
     "cssnano": "^5.1.7",
     "eslint-webpack-plugin": "^3.1.1",
+    "fast-glob": "^3.2.11",
+    "fs": "^0.0.1-security",
```

Comment:
`fs` is a core node module, no need to install this :)

Reply:
fixed in https://github.com/10up/10up-toolkit/pull/195/commits/ff2c020778d68e7955793056c2f034ac5629423d